rhino/src/main/java/org/mozilla/javascript/ScriptRuntime.java

```diff
@@ -1076,7 +1076,8 @@ public static String toString(double val) {
 
     public static String numberToString(double d, int base) {
         if ((base < 2) || (base > 36)) {
-            throw Context.reportRuntimeErrorById("msg.bad.radix", Integer.toString(base));
+            // throw Context.reportRuntimeErrorById("msg.bad.radix", Integer.toString(base));
```

Comment:
i think we can remove the throw (at lease if there is somewhere a test for that)

Reply:
Looks like a miss in my editing or merging -- this should just throw a range error. Thanks!